File: hackaton-be/src/main/java/com/purpleteam/hackaton/dto/UserAffordabilityDTO.java
package com.purpleteam.hackaton.dto;

public class UserAffordabilityDTO {

	private Double generatedMaximumAffordability;

	private Double mortgagePayment;

	private Double estimatedRemainingCash;

	private Double otherHousingCosts;

	public Double getGeneratedMaximumAffordability() {
		return generatedMaximumAffordability;
	}

	public void setGeneratedMaximumAffordability(Double generatedMaximumAffordability) {
		this.generatedMaximumAffordability = generatedMaximumAffordability;
	}

	public Double getMortgagePayment() {
		return mortgagePayment;
	}

	public void setMortgagePayment(Double mortgagePayment) {
		this.mortgagePayment = mortgagePayment;
	}

	public Double getEstimatedRemainingCash() {
		return estimatedRemainingCash;
	}

	public void setEstimatedRemainingCash(Double estimatedRemainingCash) {
		this.estimatedRemainingCash = estimatedRemainingCash;
	}

	public Double getOtherHousingCosts() {
		return otherHousingCosts;
	}

	public void setOtherHousingCosts(Double otherHousingCosts) {
		this.otherHousingCosts = otherHousingCosts;
	}

	public UserAffordabilityDTO(Double generatedMaximumAffordability, Double mortgagePayment,
			Double estimatedRemainingCash, Double otherHousingCosts) {
		super();
		this.generatedMaximumAffordability = generatedMaximumAffordability;
		this.mortgagePayment = mortgagePayment;
		this.estimatedRemainingCash = estimatedRemainingCash;
		this.otherHousingCosts = otherHousingCosts;
	}

	public UserAffordabilityDTO() {
		super();
		// TODO Auto-generated constructor stub
	}

}

File: mortgage-app/src/app/dtoModels/user-affordability-dto.ts
export class UserAffordabilityDTO{
    userId: string;
    annualIncome: number;
    monthlyHousingCosts: number;
    monthlyDebtPayments: number;
    generatedMaximumAffordability: number;
    mortgagePayment: number;
    estimatedRemainingCash: number;

    constructor(userId: string, annualIncome: number, monthlyHousingCosts: number, monthlyDebtPayments: number, generatedMaximumAffordability: number,
                mortgagePayment: number, estimatedRemainingCash: number){
            this.userId = userId;
            this.annualIncome = annualIncome;
            this.monthlyHousingCosts = monthlyHousingCosts;
            this.monthlyDebtPayments = monthlyDebtPayments;
            this.generatedMaximumAffordability = generatedMaximumAffordability;
            this.mortgagePayment = mortgagePayment;
            this.estimatedRemainingCash = estimatedRemainingCash;
    }
}